packages/editor/src/components/post-featured-image/index.js

```diff
@@ -95,6 +95,27 @@ function PostFeaturedImage( {
 		<PostFeaturedImageCheck>
 			{ noticeUI }
 			<div className="editor-post-featured-image">
+				{ media && (
+					<div
+						id={ `editor-post-featured-image-${ featuredImageId }-describedby` }
+						className={ 'hidden' }
```

Comment:
I think this doesn't need the brackets and can simply be `className="hidden"`

Reply:
Fixed :+1: